File: api/controllers/system.js
var sequence = require('sequence');
var async = require('async');
var _ = require('underscore');

module.exports = {

  'GET /system/utc_timestamp':
  // get unix timestamp with milliseconds
  //  returns time
  function($) {
    return $.render(Date.now() / 1000);
  }

};

File: api3/api/system.rb
module Piecemaker

  class System < Grape::API

    #===========================================================================
    resource 'system' do #======================================================
    #===========================================================================


      #_________________________________________________________________________
      ##########################################################################
      desc "get unix timestamp with milliseconds"
      #-------------------------------------------------------------------------
      get "/utc_timestamp" do  #/api/v1/system/utc_timestamp
      #-------------------------------------------------------------------------
        return {
          :utc_timestamp => Time.now.utc.to_f
        }
      end

    end

  end
end